WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt

```diff
@@ -176,6 +163,42 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
                 }
             }
         }
+
+        @ExperimentalStdlibApi
+        private fun showSubtitle(inputDate: Date, author: String?, subtitle: Int?) {
+            val date = if (inputDate == Date(0)) Date() else inputDate
+            val stringDate = DateTimeUtils.javaDateToTimeSpan(date, parent.context)
+                    .capitalizeWithLocaleWithoutLint(parent.context.currentLocale)
+
+            /** The subtitle can use 2 or 3 placeholders
+            * Date - Only (author & subtitle are null)
+            * Date - Author (author != null && subtitle == null)
+            * Date - subtitle (author == null && subtitle != null)
+            * Date - Author - subtitle (all have values)
+            */
+            pageSubtitle.text = if (author == null && subtitle == null) {
+                stringDate
+            } else if (author != null && subtitle == null) {
+                String.format(
+                        Locale.getDefault(),
+                        parent.context.getString(R.string.pages_item_subtitle),
+                        stringDate,
+                        author)
+            } else if (author == null && subtitle != null) {
+                String.format(
+                        Locale.getDefault(),
+                        parent.context.getString(R.string.pages_item_subtitle),
+                        stringDate,
+                        parent.context.getString(subtitle))
+            } else {
+                String.format(
+                        Locale.getDefault(),
+                        parent.context.getString(R.string.pages_item_subtitle_date_author),
+                        stringDate,
+                        author,
+                        parent.context.getString(subtitle!!))
```

Comment:
Just checking: Is the `!!` intentional?

Reply:
Yes, it was intentional because subtitle is nullable. I'll wrap it in a let instead and commit.

Reply:
You didn't have to change it. I honestly just asked if it was a leftover or an intentional usage. Either case the let looks good, thanks!